### ISSUE TITLE
Failure with dev roxygen2

### ISSUE THREAD

**Issue by hadley** [ThinkR-open/checkhelper#86]
roxygen2 now uses messages instead of warnings so the tests fail. I would have attempted a fix, but I don't understand how your package code is generated.

```
Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Complete output:
> library(testthat)
> library(checkhelper)
> 
> test_check("checkhelper")
Package: checkpackage
Title: What the Package Does (One Line, Title Case)
Version: 0.0.0.9000
...
    ▆
 1. ├─usethis::with_project(...) at test-find_missing_values.R:68:3
 2. │ └─base::force(code)
 3. └─testthat::expect_warning(attachment::att_amend_desc(), regexp = "@return") at test-find_missing_values.R:113:5
── Failure ('test-find_missing_values.R:120:5'): find_missing_tags works ───────
`out <- expect_message(find_missing_tags(path), "my_long_fun_name_for_multiple_lines_globals")` did not produce any warnings.

[ FAIL 3 | WARN 0 | SKIP 0 | PASS 67 ]
Error: Test failures
Execution halted
```

**Comment by hadley** [comment]
Since I can't provide a fix and it's a busy time of the year, I'll push submission back to Jan 15. But please let me know if you can fix earlier or its ok to break on CRAN.

**Comment by statnmap** [comment]
Thanks for the warning.  
That's ok if it fails on CRAN. We plan to provide a fix by the end of January.

> Note that I understand how it can be confusing to mix development with {fusen} if you never used it before. However, you can always provide a fix the way you use to do in a classical package. The maintainer can then decide to re-generate it or not. The package structure keeps being a classical package structure. I'll try to make it clearer for future external contributions.
